src/data/ulift.lean

```diff
@@ -1,17 +1,104 @@
 /-
 Copyright (c) 2018 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Scott Morrison
+Authors: Scott Morrison, Jannis Limperg
 
-Transport through constant families.
+Facts about `ulift` and `plift`.
 -/
 
-universes u₁ u₂
+universes u v
 
-@[simp] lemma plift.rec.constant {α : Sort u₁} {β : Sort u₂} (b : β) :
+namespace plift
+
+variables {α β : Sort u}
```

Comment:
```suggestion
variables {α : Sort u} {β : Sort v}
```

Otherwise there is little point in having a separate `plift.map` definition, you could just use `<$>`.  (To explain: the functor type-class is not universe-polymorphic enough.  `<$>` requires that the domain and codomain live in the same universe, while `plift.map` could be more general and have two universe parameters.)

Reply:
Good point, thank you. Same for `ulift` below.